src/components/Cart/CartItmCard.js

```diff
@@ -0,0 +1,107 @@
+import { useCart, useTheme } from "../../contexts"
+import { getBorderColor, getTextColor } from "../../utils"
+import { Button, Card, Image, Input, Text } from "../Reusable"
+
+const CartItmCard = ({ item: { id, name, company, img: { srcSet, alt, sizes }, quantity, size, price, offerPrice } }) => {
+    const { cartState, cartDispatch } = useCart()
+    const { theme } = useTheme()
+
+    function setQuantity(e) {
+        cartDispatch({
+            type: 'SET_QUANTITY', payload: {
+                id: id,
+                quantity: Number(e.target.value)
+            }
+        })
+    }
+
+    function setSize(e) {
+        cartDispatch({
+            type: 'SET_SIZE', payload: {
+                id: id,
+                size: Number(e.target.value)
+            }
+        })
+    }
+
+    function increaseItemQuantity() {
+        cartDispatch({ type: 'INCREASE_QUANTITY', payload: id })
+    }
+
+    function decreaseItemQuantity() {
+        cartDispatch({ type: 'DECREASE_QUANTITY', payload: id })
+    }
+
+    function removeItemFromCart() {
+        cartDispatch({ type: 'REMOVE_ITEM', payload: id })
+    }
+
+    return (
+
+        <Card classes='flx flx-row mg-btm-s'>
+
+            <Image srcSet={srcSet} alt={alt} sizes={sizes} />
+
+            <div className="flx-grow-1 pd-left-s pd-right-s pd-top-xs">
+
+                <div className="flx">
+
+                    <Text classes='txt-md txt-ucase mg-right-s'>{company}</Text>
+
+                    <Text classes='txt-md txt-cap'>{name}</Text>
+
+                </div>
+
+                <Text classes='txt-md txt-ucase mg-top-s'>{`UK ${size}`}</Text>
+
+                {/* after merging run a map on productsData.size */}
+                <select onChange={(e) => setSize(e)}>
+                    <option value='9'>9</option>
+                    <option value='10'>10</option>
+                    <option value='11'>11</option>
+                </select>
```

Comment:
appreciation - good to see that you are adding size selection also

Reply:
thank you.